scripts/aquatic_animal_excretion.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: latin-1 -*-
```

Comment:
This `latin-1` should change to UTF-8

latin cannot get the character `ä `in citation `Uye, S.-I., Ventelä, A.-M., Villeger, S., Whiles, `